pom.xml

```diff
@@ -304,7 +304,7 @@
                             <sourcepath>${project.build.sourceDirectory}:${project.build.directory}/generated-sources/annotations</sourcepath>
                             <show>public</show>
                             <failOnError>true</failOnError>
-                            <failOnWarnings>true</failOnWarnings>
+                            <failOnWarnings>false</failOnWarnings>
```

Comment:
This sucks because it disables Javadocs checks for all other classes as well.

Reply:
I know
But in case of the Builder, Fluent and FluentImpl classes, there are Javadoc warnings that will fail the build.
I was thinking about having the separate module just for these resources (the `api` module) and then the operator module, to split the things and manipulate it better in the children pom.xmls

Reply:
@scholzj @katheris @mstruk WDYT about having two modules and splitting the models with the operator's code?

Reply:
IIUC separating out the `api` module would be like the Strimzi operator project does it. Also, I guess we need it this way in order to have two different warning treatments. Sounds to me like the way to go.

Reply:
I prefer the suggestion of having a separate `api` module similar to the cluster-operator. I guess it also distinguishes which classes we might want to encourage users to depend on if they were programmatically creating `KafkaAccess` resources

Reply:
Cool, so I will add the `api` module there :) thanks